packages/mobile/src/config.ts

```diff
@@ -57,6 +61,8 @@ export const FAQ_LINK = 'https://celo.org/faq'
 export const FORUM_LINK = 'https://forum.celo.org/c/support'
 export const CELO_SUPPORT_EMAIL_ADDRESS = 'support@celo.org'
 export const DEFAULT_FORNO_URL = `https://${DEFAULT_TESTNET}-forno.celo-testnet.org`
+export const SIGN_MOONPAY_FIREBASE_URL =
+  'https://us-central1-celo-org-mobile.cloudfunctions.net/signMoonpay'
```

Comment:
Should this vary for different environments? If so, maybe move it to `network-config.ts`

Reply:
Good idea!